docs/core/reserved_verbs.md

```diff
@@ -0,0 +1,26 @@
+There are a couple of reserved verbs which ONEm platform implements. These cannot be used or overridden and will automatically work in your micro-app.
+
+| Verb Name | Notes |
+|-----------|-------|
+| back     | Allows the user to go back to the previous screen |
+| credits  | Shows app credits |
+| go       | Shows a specific chunk, e.g: `go 4`, shows 4th chunk |
+| help     | Shows app help |
+| menu     | Shows app home menu |
+| mode     | Shows the current app |
+| more     | Goes to next available chunk |
+| options  | Shows the current options |
+| size     | Sets the sms size |
+| tips     | Toggles footers |
+| tutorial | Shows app tutorial |
```

Comment:
We need to decide on these reserved verbs how to implement `help` and `tutorial` 

An easy way for us is to "unreserve" them .. meaning we take them out from the verbs category and let the developer implement them. What do you think?

Reply:
Can be the following an option?

Let the dev implement any word he wants, and if it's a generic verb, let's say help, apply this behavior when the user sends "help":

- If the current mode has `help` verb, platform replies:
```
Did you mean
A. ONEm help
B. #CURRENT-MODE help
```
- If no current mode or current mode has no `help` verb, defined platform replies with ONEm help content.

Reply:
We cannot allow the dev to implement ANY verb it wants .. For example menu performs specific actions, cache wise and the cache state can get out of sync if we allow the dev to implement menu. Or go/more .. which are chunking related.

Help and tutorial .. this is more app centric, we could do a did you mean, but what would we show if the user chooses `onem help/tutorial` ?

Reply:
You are right, not ANY verb.

b Help and tutorial .. this is more app centric, we could do a did you mean, but what would we show if the user chooses onem help/tutorial ?
What we already show, or something else. The only important thing I see here is to try to help the user if types "help". IDK.

Reply:
We have a general help wizard, with info on how to use the platform. That could be a last resort if the dev wont implement custom help.

I would completely discard tutorial ..